packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -168,6 +172,27 @@ function createResourceTimingSpans(entries, filterUrls) {
   return spans
 }
 
+function createUserTimingSpans(entries) {
+  const userTimingSpans = []
+
+  for (let i = 0; i < entries.length; i++) {
+    const { name, startTime, duration } = entries[i]
+
+    if (duration <= USER_TIMING_THRESHOLD) {
+      continue
+    }
+    const kind = 'user.measure'
```

Comment:
Trying to figure out if this type and subType info for User timing makes sense, Thoughts? @jahtalab

Reply:
Since its part of the application code, I am learning towards keeping type as `app.custom`  or `custom.app`. All the custom instrumentations in other agents use something similar with type set to custom.

Reply:
Decided to set it to `app` for now, we can add subType later if needed!